src/js/state/createGlobalStore.js

```diff
@@ -1,9 +1,12 @@
 /* @flow */
 import {createStore} from "redux"
 
+import {isCurrentVersion} from "../initializers/initPersistance"
 import globalReducer, {type GlobalState} from "./globalReducer"
 
-export default function(initialState: GlobalState | void) {
+export default function(prevState: GlobalState | void) {
+  let initState = isCurrentVersion(prevState) ? prevState : undefined
```

Comment:
I figure it was best to let each store creation be responsible for dealing with its version.